packages/source-fetcher/lib/etherscan.ts

```diff
@@ -91,7 +91,7 @@ const EtherscanFetcher: FetcherConstructor = class EtherscanFetcher
 
   private async makeRequest(address: string): Promise<EtherscanSuccess> {
     //not putting a try/catch around this; if it throws, we throw
-    const response: AxiosResponse<EtherscanResponse> = await axios.get(
+    const response = (await axios.get(
```

Comment:
A small nitpick but should we rename this to something that indicates that it is not the response and rather the payload? Either way I'm fine with it.

Reply:
I think the name's fine.  It is a response, just not in the sense you're talking about.  I should probably add the type annotations back in though; I'll go do that.